status/k8s.go

```diff
@@ -357,13 +358,28 @@ func (k *K8sStatusCollector) Status(ctx context.Context) (*Status, error) {
 		}
 		if !k.statusIsReady(s) && k.params.Wait {
 			time.Sleep(defaults.WaitRetryInterval)
+			if k.params.Output != "json" {
```

Comment:
I think we should use the predefined const. Also perhaps best to check `summary` vs `json` in case other fmts are exposed in the future.

Reply:
Yes. The const is in a package I can't import though, because that creates a dependency loop. Should we move the const to another package?